Cargo.toml

```diff
@@ -91,5 +91,6 @@ exclude = [
     "programs/bpf",
 ]
 
-# This prevents a Travis CI error when building for Windows.
-resolver = "2"
+# for details, see https://github.com/ryoqun/crossbeam/commit/fd279d707025f0e60951e429bf778b4813d1b6bf
```

Comment:
for convenience: https://github.com/ryoqun/crossbeam/commit/fd279d707025f0e60951e429bf778b4813d1b6bf